package.json

```
@@ -19,16 +19,16 @@
     ]
   },
   "dependencies": {
-    "@hapi/boom": "9.x.x",
-    "@hapi/bounce": "2.x.x",
-    "@hapi/bourne": "2.x.x",
-    "@hapi/hoek": "9.x.x",
-    "@hapi/validate": "1.x.x"
+    "@hapi/boom": "^10.0.0",
+    "@hapi/bounce": "^3.0.0",
+    "@hapi/bourne": "^3.0.0",
+    "@hapi/hoek": "^10.0.0",
+    "@hapi/validate": "^1.0.0"
```

Pending https://github.com/hapijs/validate/pull/24
```suggestion
    "@hapi/validate": "^2.0.0"
```